beep/protocol/biologic.py

```diff
@@ -164,3 +163,129 @@ def to_file(self, filename, encoding="ISO-8859-1", column_width=20, linesep="\r\
 
         with open(filename, "wb") as f:
             f.write(contents.encode(encoding))
+
+    def formation_protocol_bcs(self, params):
```

Comment:
is bcs an extension for one of biologic output files?

Reply:
boundary conditions? :-)

Reply:
haha, I think that bcs is Biologic cycler system? Not sure actually :P